java/client/src/main/java/glide/api/models/commands/BitcountOptions.java

```diff
@@ -0,0 +1,29 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.models.commands;
+
+import glide.api.commands.BitmapBaseCommands;
+import lombok.RequiredArgsConstructor;
+
+/**
+ * Optional arguments for {@link BitmapBaseCommands#bitcount(String, long, long, BitcountOptions)}.
+ *
+ * @see <a href="https://redis.io/commands/bitcount/">redis.io</a>
+ */
+@RequiredArgsConstructor
+public enum BitcountOptions {
+    /** By default, the additional arguments start and end specify a byte index */
```

Comment:
Don't copy redis docs and don't describe default behaviour
As I undestand, this parameter specifies whether indices given in the command args are bit indices or byte indices. That's correct? Try to reword the doc.

Reply:
Thanks! Addressed in https://github.com/Bit-Quill/glide-for-redis/pull/263/commits/2575b0125be5640b722ee685c33c9fd624ae1b00